spec/requests/api/departments/index_spec.rb

```diff
@@ -18,6 +18,10 @@
     it 'is expected to have a participant' do
       expect(response_json['departments'].first['participants'].first['name']).to eq participant.name
     end
+
+    it 'is expected to have a participants avatar' do      
+      expect(response_json['departments'].first['participants'].first).to include "avatar"
```

Comment:
Format for single quotations and also in the create_spec below.

Reply:
This has been changed